kodilla-spring/src/main/java/com/kodilla/spring/calculator/Display.java

```diff
@@ -0,0 +1,11 @@
+package com.kodilla.spring.calculator;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class Display {
+
+    public double displayValue(double val) {
+        return val;
```

Comment:
Jak sama nazwa wskazuje, to ma cos wyswietlac.
public void displayValue(double val) {
   sout(val);
}